Source/Hurl.App/MainWindow.xaml

```diff
@@ -0,0 +1,20 @@
+﻿<ui:FluentWindow
+    x:Class="Hurl.App.MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="clr-namespace:Hurl.App"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:ui="http://schemas.lepo.co/wpfui/2022/xaml"
+    Title="MainWindow"
+    Width="800"
+    Height="450"
+    ExtendsContentIntoTitleBar="True"
+    WindowBackdropType="Mica"
+    WindowCornerPreference="Round"
+    WindowStartupLocation="CenterScreen"
+    mc:Ignorable="d">
+    <Grid>
+        <ui:TitleBar Title="Hurl [BETA]" Grid.ColumnSpan="3" />
```

Comment:
Urgently recommend **not** to use the same Grid idea

Reply:
Why?

Reply:
Because it makes everything way too complicated. You need to remember to put everything in the middle, and you will never have a scrollviewer that touches the sides. The same effect can be achieved with MaxWidth of the content.